src/SignalR/common/Http.Connections/src/Internal/HttpConnectionDispatcher.cs

```diff
@@ -26,21 +26,21 @@ internal partial class HttpConnectionDispatcher
             new AvailableTransport
             {
                 Transport = nameof(HttpTransportType.WebSockets),
-                TransferFormats = new List<string> { nameof(TransferFormat.Text), nameof(TransferFormat.Binary) }
+                TransferFormats = new List<string>(2) { nameof(TransferFormat.Text), nameof(TransferFormat.Binary) }
```

Comment:
Could these be arrays instead of the list?

Reply:
I guess it depends on whether the intention is for them to be fixed-size. They _probably_ are, but I didn't want to change _too_ much beyond the initial scope unless asked.